src/components/index/RequestsTableWithPagination.tsx

```diff
@@ -13,12 +15,14 @@ export const RequestsTableWithPagination = ({
   setCurrentPage,
   requests,
 }: Props) => {
+  const [dropdownValue, setDropdownValue] = useState<OptionType>(items[0]);
```

Comment:
this should be hoisted or you will lose the elementsPerPage when you visit a particular request and come back to index